content/en/docs/Reference/admission-webhooks.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Admission Webhook Reference"
+linkTitle: "Admission Webhook Reference"
+weight: 3
+date: 2020-04-24
+---
+
+After a request has been authenticated and authorized, admission webhooks intercept requests against the Kubernetes API and have an opportunity to validate or update the object before it is saved in the object store. Please refer to the following table that highlights what each webhook is capable of:
```

Comment:
Seems a little odd that we're making a reference section that isn't just a list of one-liner descriptions with links.

Also, I can't see where this is referenced anywhere but the reference file.

Reply:
This reference doc was built [based on input](https://github.com/operator-framework/olm-docs/pull/17#discussion_r417543141) from @anik120. The file is linked [on line 11](https://github.com/operator-framework/olm-docs/pull/17/files#diff-a6d844b8a6bf3a81d5966cee50d09408R11) of admission-webhooks.md.

Given that there is only one reference, I can just update the admission-webhooks.md file to just include a link to the official k8s documentation if you would like.

Reply:
Let's leave this alone for now since I think we'll be revising the structure anyway. The only thing I think we should do in this PR is remove the "Coming Soon" banner from the references page.